__tests__/app.test.js

```diff
@@ -20,14 +20,29 @@ describe('GET /api/topics task 3',() => {
                     expect.objectContaining({
                       slug: expect.any(String),
                       description: expect.any(String),
-                    })
-                  );
-                // expect.objectContaining({
-                //     'slug': expect(typeof(topicObj['slug'])).toBe('string'),
-                //     'description': expect(typeof(topicObj['description'])).toBe('string')
-                // })
-                // expect(Object.keys(obj)).toMatchObject(['description','slug'])
-            })
+                    }));
+            });
         });
     });
 });
+
+
+//4
+describe.only('GET /api/articles task 4',() => {
+    test('responds with status code 200, appropriate object properties and values',() => {
+        return request(app).get('/api/articles')
+        .expect(200)
+        .then(({body}) => {
+            expect(Array.isArray(body.articles)).toBe(true);
```

Comment:
We don't need to explicitly test for an array here as we are using a forEach afterwards which proves that is an array. A better test may be to check the length?